test/models/ping_test.rb

```diff
@@ -164,7 +164,6 @@ def test_ping_katello_agent_enabled
 
       assert_equal 'ok', result[:status]
       assert_equal '0 Processed, 0 Failed', result[:message]
-      assert_equal '0', result [:duration_ms]
```

Comment:
Does it make sense to assert it's present?
```ruby
assert_includes(result, :duration_ms)
```

Reply:
maybe? but we don't do that in the other tests either…

Reply:
Perfectly find without it imo.